<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawDataRiverSurveyCrosssectionWindow</class>
 <widget class="QMainWindow" name="RawDataRiverSurveyCrosssectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="RawDataRiverSurveyCrosssectionWindowGraphicsView" name="graphicsView" native="true"/>
      <widget class="QSplitter" name="verticalSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="surveysTableWidget"/>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="weCheckBox">
             <property name="text">
              <string>Specify water elevation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="weSpinBox">
             <property name="minimum">
              <double>-10000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="RawDataRiverSurveyCrosssectionWindowTableView" name="tableView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RawDataRiverSurveyCrosssectionWindowGraphicsView</class>
   <extends>QWidget</extends>
   <header>rawdatariversurveycrosssectionwindowgraphicsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RawDataRiverSurveyCrosssectionWindowTableView</class>
   <extends>QTableView</extends>
   <header>rawdatariversurveycrosssectionwindowtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
